android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -427,6 +437,23 @@ private boolean removeOrAnimateExitRecursive(View view, ViewGroup parent, boolea
     return true;
   }
 
+  public void clearAnimationConfigForTag(int tag) {
+    View view = resolveView(tag);
+    if (view != null) {
+      clearAnimationConfigRecursive(view);
+    }
+  }
+
+  public void clearAnimationConfigRecursive(View view) {
+    mNativeMethodsHolder.clearAnimationConfig(view.getId());
+    if (view instanceof ViewGroup) {
+      ViewGroup viewGroup = (ViewGroup) view;
+      for (int i = 0; i < viewGroup.getChildCount(); i++) {
```

Comment:
What if some screens are removed from native hierarchy? Maybe we should use React's tree instead of. (I just wondering)